confluent_kafka/src/confluent_kafka.c

```diff
@@ -1444,7 +1444,7 @@ static int producer_conf_set_special (Handle *self, rd_kafka_conf_t *conf,
 				      rd_kafka_topic_conf_t *tconf,
 				      const char *name, PyObject *valobj) {
 
-	if (!strcasecmp(name, "on_delivery")) {
+	if (!strcmp(name, "on_delivery")) {
```

Comment:
Is there an `strcasecmp` equivalent for windows ? Otherwise `tolower()` or some equivalent  seem appropriate prior to comparison. While I agree function arguments should be case-sensitive they haven't been in the past so doesn't this technically break the api?

Reply:
There are Windows alternatives, but since python is case-sensitive, and we haven't made any claims that on_delivery, et.al, may be specified in non lower-case, I think it is safe to remove this "feature".

Reply:
If you're good with it I'm good with it

Reply:
Then I'm good with it